trino/sqlalchemy/dialect.py

```diff
@@ -280,7 +280,11 @@ def get_indexes(self, connection: Connection, table_name: str, schema: str = Non
         if not self.has_table(connection, table_name, schema):
             raise exc.NoSuchTableError(f"schema={schema}, table={table_name}")
 
-        partitioned_columns = self._get_columns(connection, f"{table_name}$partitions", schema, **kw)
+        partitioned_columns = None
+        try:
+            partitioned_columns = self._get_columns(connection, f"{table_name}$partitions", schema, **kw)
+        except Exception as e:
+            logger.debug("Couldn't fetch partition columns. table: %s, schema: %s, error: %s", table_name, schema, e)
```

Comment:
We usually order identifiers as schema → table. 
```suggestion
            logger.debug("Couldn't fetch partition columns. schema: %s, table: %s, error: %s", schema, table_name, e)
```

Reply:
fixed!